src/TrueLayer/Payments/IPaymentsApi.cs

```diff
@@ -81,5 +84,40 @@ public interface IPaymentsApi
             StartAuthorizationFlowRequest request,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Create a partial or full refund for a payment.
+        /// </summary>
+        /// <param name="paymentId">The payment identifier</param>
+        /// <param name="idempotencyKey">
+        /// An idempotency key to allow safe retrying without the operation being performed multiple times.
+        /// The value should be unique for each operation, e.g. a UUID, with the same key being sent on a retry of the same request.
+        /// </param>
+        /// <param name="request">The create payment refund request</param>
+        /// <param name="cancellationToken">The cancellation token to cancel the operation</param>
+        /// <returns>The id of the created refund</returns>
+        Task<ApiResponse<CreatePaymentRefundResponse>> CreatePaymentRefund(string paymentId,
+            string idempotencyKey,
+            CreatePaymentRefundRequest request,
+            CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Get the list of all refunds for a payment.
+        /// </summary>
+        /// <param name="paymentId">The payment identifier</param>
+        /// <param name="cancellationToken">The cancellation token to cancel the operation</param>
+        /// <returns>The list of refunds for a payment.</returns>
+        Task<ApiResponse<ListPaymentRefundsResponse>> ListPaymentRefunds(string paymentId,
```

Comment:
Can you please rename it to match how the endpoint is referenced in the API ref (`GetPaymentRefunds`)?

Reply:
Never mind, I can do it in my PR